packages/orbit-components/src/InputFile/InputFile.stories.tsx

```diff
@@ -1,208 +1,121 @@
 import * as React from "react";
+import type { Meta, StoryObj } from "@storybook/react";
 import { action } from "@storybook/addon-actions";
 
 import RenderInRtl from "../utils/rtl/RenderInRtl";
 import { SPACINGS_AFTER } from "../common/consts";
 
 import InputFile from ".";
 
-export default {
+const meta: Meta<typeof InputFile> = {
   title: "InputFile",
-};
-
-export const Default = ({ label }) => {
-  return (
-    <InputFile
-      label={label}
-      onChange={action("onChange")}
-      onFocus={action("onFocus")}
-      onBlur={action("onBlur")}
-      onRemoveFile={action("removeFile")}
-    />
-  );
-};
+  component: InputFile,
 
-Default.story = {
   parameters: {
-    info: "You can try all possible configurations of this component. However, check Orbit.Kiwi for more detailed design guidelines.",
+    controls: {
+      exclude: ["onChange", "onFocus", "onBlur", "onRemoveFile"],
+    },
   },
-};
 
-Default.args = {
-  label: "Label",
+  args: {
+    label: "Label",
+    onChange: action("onChange"),
+    onFocus: action("onFocus"),
+    onBlur: action("onBlur"),
+    onRemoveFile: action("removeFile"),
+  },
 };
 
-export const FilledWithFile = ({ label, fileName }) => {
-  return (
-    <InputFile
-      label={label}
-      fileName={fileName}
-      onChange={action("onChange")}
-      onFocus={action("onFocus")}
-      onBlur={action("onBlur")}
-      onRemoveFile={action("removeFile")}
-    />
-  );
-};
+export default meta;
+type Story = StoryObj<typeof InputFile>;
 
-FilledWithFile.story = {
-  name: "Filled with file",
+export const Default: Story = {};
 
-  parameters: {
-    info: "You can try all possible configurations of this component. However, check Orbit.Kiwi for more detailed design guidelines.",
+export const FilledWithFile: Story = {
+  args: {
+    fileName: "file.png",
   },
 };
 
-FilledWithFile.args = {
-  label: "Label",
-  fileName: "file.png",
-};
-
-export const WithHelp = ({ label }) => {
-  return (
-    <InputFile
-      label={label}
-      help={
-        <div>
-          Supported files: <strong>PNG, JPG, PDF</strong>
-        </div>
-      }
-      onChange={action("onChange")}
-      onFocus={action("onFocus")}
-      onBlur={action("onBlur")}
-      onRemoveFile={action("removeFile")}
-    />
-  );
-};
-
-WithHelp.story = {
-  name: "With help",
-
+export const WithHelp: Story = {
   parameters: {
-    info: "You can try all possible configurations of this component. However, check Orbit.Kiwi for more detailed design guidelines.",
+    controls: {
+      exclude: ["onChange", "onFocus", "onBlur", "onRemoveFile", "help"],
```

Comment:
Unfortunately, yes 😞  It's not very clear in the SB docs, but I found the response on the SB Discord that it will always override the meta excluded values and we need to provide a complete value for each override, there are two ways.

We can do it either like I did or we can spread the meta values but with disabling ESlint as well (or we could add  there`|| {}` to fix the error)

```
  parameters: {
    controls: {
      // eslint-disable-next-line no-unsafe-optional-chaining
      exclude: [...meta.parameters?.controls.exclude, "help"],
    },
  },
```

Which one do you prefer? To me the least complicated one is to just enumerate it again. But with the spread it at least provides some context that we are aware of the meta excludes and we need to extend it.